app/helpers/application_helper/button/csv.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationHelper::Button::Csv < ApplicationHelper::Button::Basic
+  def check_rbac?
+    %w(render_report_csv chargeback_download_csv compare_download_csv).include?(self[:child_id]) ? role_allows?(:feature => self[:child_id]) : true
```

Comment:
Question, is there an intention to remove these data clump arrays into one or two class constants?  I'd imagine if we need to add a new action, we'd have to put `my_new_action_csv`, `my_new_action_pdf`, `my_new_action_txt` into at least 3 different arrays split through 3 different files.